r/R/type.R

```diff
@@ -180,15 +180,30 @@ NestedType <- R6Class("NestedType", inherit = DataType)
 #' types, this conversion can be disabled (so that `int64` always yields a
 #' `bit64::integer64` object) by setting `options(arrow.int64_downcast =
 #' FALSE)`.
+#' `decimal()` creates a `decimal128` type. Arrow decimals are fixed-point
```

Comment:
Add newline before this?

Reply:
Sure. I'm actually adding a bit more to that section. Will push shortly